client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -75,6 +96,55 @@ const productSaleModel = {
 			},
 		}
 	},
+	conditions: {
+		labelText: translate( 'Conditions', { context: 'noun' } ),
+		fields: {
+			startDate,
+			endDate,
+		}
+	},
+};
+
+/**
+ * Conditions for all coupon types.
+ */
+const couponConditions = {
+	labelText: translate( 'Conditions', { context: 'noun' } ),
+	fields: {
+		endDate,
+		minimumAmount: {
+			component: CurrencyField,
+			labelText: translate( 'This promotion requires a minimum purchase' ),
+			isEnableable: true,
+			defaultValue: 10,
+		},
+		maximumAmount: {
+			component: CurrencyField,
+			labelText: translate( 'The total discount cannot exceed a certain amount' ),
```

Comment:
@kellychoffman I don't think this is the right wording. I think this is actually a maximum spend of sorts. I don't know what the actual code does, but I think it probably limits the maximum benefit of the coupon to this spending limit. Here's what wp-admin says about it:

![screen shot 2017-10-30 at 10 40 18 pm](https://user-images.githubusercontent.com/945228/32206508-83fbc23a-bdc3-11e7-9847-197f366e521d.png)